src/Model/TransportModel/tsp1apt1.f90

```diff
@@ -1068,7 +1039,8 @@ subroutine apt_ot_dv(this, idvsave, idvprint)
         end if
         this%dbuff(n) = c
       end do
-      call ulasav(this%dbuff, '   CONCENTRATION', kstp, kper, pertim, totim, &
+      write (text, '(a)') str_pad_left(this%depvartype, 16)
```

Comment:
Is there a way to avoid hard coding this "16"?  Should this be one of our constants?